acceptance/detector_test.go

```diff
@@ -0,0 +1,209 @@
+			h.AssertStringContains(t, string(groupContents), expected)
+
+			// check plan.toml - should be empty
+			tempPlanToml, err := ioutil.TempFile("", "")
+			h.AssertNil(t, err)
+			defer os.RemoveAll(tempPlanToml.Name())
+
+			h.Run(t, exec.Command(
+				"docker", "cp",
+				fmt.Sprintf("%s:/layers/plan.toml", containerName),
+				tempPlanToml.Name(),
+			))
+
+			planContents, err := ioutil.ReadFile(tempPlanToml.Name())
+			h.AssertNil(t, err)
+			h.AssertEq(t, len(planContents) > 0, true)
+			expected = "[[entries]]\n\n  [[entries.providers]]\n    id = \"simple_buildpack\"\n    version = \"simple_buildpack_version\"\n\n  [[entries.requires]]\n    name = \"some-world\"\n    version = \"0.1\"\n    [entries.requires.metadata]\n      world = \"Earth-616\""
+			h.AssertStringContains(t, string(planContents), expected)
+		})
+	})
+
+	when("environment variables are provided for buildpack and app directories and for the output files", func() {
+		it("writes group.toml and plan.toml in the right location and with the right names", func() {
+			containerName := "test-container-" + h.RandString(10)
+			h.Run(t, exec.Command(
+				"docker",
+				"run",
+				"--name", containerName,
+				"--user", userID,
+				"--env", "CNB_ORDER_PATH=/cnb/orders/always_detect_order.toml",
+				"--env", "CNB_BUILDPACKS_DIR=/cnb/custom_buildpacks",
+				"--env", "CNB_APP_DIR=/custom_workspace",
+				"--env", "CNB_GROUP_PATH=./custom_group.toml",
+				"--env", "CNB_PLAN_PATH=./custom_plan.toml",
+				"--env", "CNB_PLATFORM_DIR=/custom_platform",
+				detectImage,
+				"-log-level=debug"))
+			defer h.Run(t, exec.Command("docker", "rm", containerName))
+
+			// check group.toml
+			tempGroupToml, err := ioutil.TempFile("", "")
+			h.AssertNil(t, err)
+			defer os.RemoveAll(tempGroupToml.Name())
+
+			h.Run(t, exec.Command(
+				"docker", "cp",
+				fmt.Sprintf("%s:/layers/custom_group.toml", containerName),
+				tempGroupToml.Name(),
+			))
+
+			groupContents, err := ioutil.ReadFile(tempGroupToml.Name())
+			h.AssertNil(t, err)
+			h.AssertEq(t, len(groupContents) > 0, true)
+			expectedGroupContents := "[[group]]\n  id = \"always_detect_buildpack\"\n  version = \"always_detect_buildpack_version\""
+			h.AssertStringContains(t, string(groupContents), expectedGroupContents)
+
+			// check plan.toml - should be empty
+			tempPlanToml, err := ioutil.TempFile("", "")
+			h.AssertNil(t, err)
+			defer os.RemoveAll(tempPlanToml.Name())
+
+			h.Run(t, exec.Command(
+				"docker", "cp",
+				fmt.Sprintf("%s:/layers/custom_plan.toml", containerName),
+				tempPlanToml.Name(),
+			))
+
+			planContents, err := ioutil.ReadFile(tempPlanToml.Name())
+			h.AssertNil(t, err)
+			h.AssertEq(t, len(planContents) == 0, true)
+
+			// check platform directory
+			containerId := strings.TrimSuffix(h.Run(t, exec.Command("docker", "ps", "-aqf", fmt.Sprintf("name=%s", containerName))),"\n")
+			logs := h.Run(t, exec.Command("docker", "logs", containerId))
```

Comment:
```suggestion
			logs := h.Run(t, exec.Command("docker", "logs", containerName))
```
This should work with the container name, no need to grab the `containerId`.

Reply:
Oops. You're right :)